tests/console/yast2_bootloader.pm

```diff
@@ -25,7 +25,6 @@ sub run() {
     script_run("/sbin/yast2 bootloader; echo YBL-$? > /dev/$serialdev", 0);
     my $ret = assert_screen "test-yast2_bootloader-1", 300;
```

Comment:
As you are hunting for redundancies you may want to look for unused code too. `$ret` is prime example for removal.